nix/default.nix

```
@@ -61,6 +61,7 @@ in
   (lib.checkListOfEnum "${pname}: bracketMode" validBracketModes [bracketMode])
   (pkgs.vscode-utils.buildVscodeExtension {
     inherit name version vscodeExtPublisher vscodeExtName vscodeExtUniqueId;
+    pname = name;
```

`pname` exists within the `let in` block, so lets inherit that and remove the `name`

```suggestion
    inherit pname version vscodeExtPublisher vscodeExtName vscodeExtUniqueId;
```

---

The `pname` in the scope contains both the `name` and `version`. I think pname isn't suppose to contain the version. It will result in a path like `/nix/store/m19p8jmll28lrfaczb4liai4iwv24rd9-vscode-extension-catppuccin-vsc-3.16.0-3.16.0.drv`.

I'll try to fix the `pname` in the scope.